wetterdienst/indexing/file_index_creation.py

```diff
@@ -26,7 +29,7 @@
 )
 
 
-@lru_cache(maxsize=None)
+@fileindex_cache_five_minutes.cache_on_arguments()
```

Comment:
Is there a reason for using 5 minutes here and for RADOLAN one hour?

Reply:
Not really. I believed it would make sense to lower the ttl here in order to get fresh data when aiming for `MINUTES_10`. Should we also use a lower value for RADOLAN? Actually, I didn't read upfront about the refresh constraints, maybe you can guide me here?

Reply:
I just checked it and believe that 10_minutes and 1_minute is only updated very hour and all lower resolution data is only updated daily

Reply:
But what if the user requests something e.g. five minutes before a new artefact will be published? Will the machinery miss that file for another half an hour?

Reply:
That is true and 10 seconds or so that it takes to build the (biggest) index is comparatively low compared to 300 seconds.

Reply:
Sure. So, let's keep it like it is and just cache it for just five minutes?